module/Application/src/Application/Session/SaveHandler/EncryptedSessionSaveHandler.php

```diff
@@ -81,6 +75,7 @@ public function destroy($id)
      * @param int $maxlifetime
      * @return bool
      */
+    #[ReturnTypeWillChange]
```

Comment:
would it be better to change this to not return the value since we don't use it, and have the return type as `void`